server/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "private": true,
   "scripts": {
-    "start": "node index.js",
-    "dev": "nodemon --exec npm start"
+    "start": "node generateLeafletMap.js && node index.js",
+    "dev": "node generateLeafletMap.js && nodemon --exec npm start"
   },
   "keywords": [],
   "author": "",
```

Comment:
Could do with `"map/leaflet-map",` adding to the nodemon ignore list below, line 14, so it doesn't have to track the almost 3000 extra generated files.

Reply:
Yes I tried that and I came to the conclusion that this is not a good idea.
Even though it is ignored in nodemon there's a possibility that a new developer might start coding right away which then triggers a restart.
This restart could lead to problem if the  `generateLeafletMap.js` is not yet done generating map files.

Reply:
Maybe this script should only be run with a separate "build" script, separate from the development ones, so only people that care about using the map during development need to generate the tiles.